### PR TITLE
show sessions as a row

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
### problem
space_wasted_on_the_screen

### solution
screen shot 2018-02-06 at 10 32 10 am

### school view
screen shot 2018-02-06 at 10 38 05 am

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html [hunk 1]
```diff
@@ -50,14 +50,14 @@
       <div class="col-md-3">
         <div *ngIf="allowFilterBySessions">
           <p><strong>{{'labels.groups.results.assessment.sessions-title' | translate}}</strong></p>
-          <div class="btn-group-vertical">
-            <a *ngFor="let session of sessions" class="reverse multi-line btn"
+          <div class="flex-children">
+            <a *ngFor="let session of sessions" class="reverse multi-line btn btn-sm flex-child"
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: `btn-sm` does make this seem a little abnormally small, but it helps shorten the vertical distance

**Comment by jtreuting** [comment]
@aaronsleeper can you review to see what you think?

File: webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html (hunk 1 above)
Comment (aaronsleeper) [reply webapp/src/main/webapp/src/app/assessments/results/assessment-results.component.html]: @jtreuting this doesn't look like what we discussed. am i missing something?

**Comment by jtreuting** [comment]
looks good to me and is what Aaron had in mind.  since in production we do have some with a whole lot of sessions, this will help for sure.

**Pull request merged**